installation/index.md

```diff
@@ -36,13 +36,16 @@ The previously mentioned [openHABian](openhabian.html) can also be used to kicks
 
 ## Prerequisites
 
-Make sure that you have an up to date version of the [Oracle Java Runtime Environment](https://java.com/en/download/manual.jsp) on your host system installed.
-A famous choice is OpenJDK, however, Oracle Java Version 8 is recommended for openHAB.
-openHAB may be unstable and slower executed by OpenJDK.
+Make sure that you have an up to date Java Runtime Environment (JRE) installed on your host system. The [Oracle JRE](https://java.com/en/download/manual.jsp)
```

Comment:
Even without an apt-repo, it should be fairly simple to install Zulu embedded as part of openHABian initial scripts.

```bash
wget http://cdn.azul.com/zulu-embedded/bin/ezdk-1.8.0_112-8.19.0.31-eval-linux_aarch32hf.tar.gz
sudo mkdir /var/lib/zulu
sudo tar xvfpz ezdk-1.8.0_112-8.19.0.31-eval-linux_aarch32hf.tar.gz -C /var/lib/zulu
sudo update-alternatives --install /usr/bin/java java /var/lib/zulu/ezdk-1.8.0_112-8.19.0.31-eval-linux_aarch32hf/bin/java 100
sudo update-alternatives --install /usr/bin/javac javac /var/lib/zulu/ezdk-1.8.0_112-8.19.0.31-eval-linux_aarch32hf/bin/javac 100
```